test/core/vector/CMakeLists.txt

```diff
@@ -8,19 +8,24 @@ file(CREATE_LINK ${SIM_BASE}/arches     ${CMAKE_CURRENT_BINARY_DIR}/arches
-file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/undisturbed_checking.json  ${CMAKE_CURRENT_BINARY_DIR}/undisturbed_checking.json SYMBOLIC)
+file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/vsetivli_vaddvv.json      ${CMAKE_CURRENT_BINARY_DIR}/vsetivli_vaddvv.json SYMBOLIC)
+file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/vsetvli_vaddvv.json       ${CMAKE_CURRENT_BINARY_DIR}/vsetvli_vaddvv.json SYMBOLIC)
+file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/vsetvl_vaddvv.json        ${CMAKE_CURRENT_BINARY_DIR}/vsetvl_vaddvv.json SYMBOLIC)
+file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/vsetivli_vaddvv_tail.json ${CMAKE_CURRENT_BINARY_DIR}/vsetivli_vaddvv_tail.json SYMBOLIC)
+file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/multiple_vset.json        ${CMAKE_CURRENT_BINARY_DIR}/multiple_vset.json SYMBOLIC)
+file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/vmulvx.json               ${CMAKE_CURRENT_BINARY_DIR}/vmulvx.json SYMBOLIC)
+file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/vwmulvv.json              ${CMAKE_CURRENT_BINARY_DIR}/vwmulvv.json SYMBOLIC)
+file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/vmseqvv.json              ${CMAKE_CURRENT_BINARY_DIR}/vmseqvv.json SYMBOLIC)
+file(CREATE_LINK ${CMAKE_CURRENT_SOURCE_DIR}/vrgather.json             ${CMAKE_CURRENT_BINARY_DIR}/vrgather.json SYMBOLIC)
 
 
 
-sparta_named_test(Vector_test_vsetivli_lmul_4        Vector_test big_core.out -c test_cores/test_big_core_full.yaml --input-file vsetivli_vadd_lmul_4.json)
-sparta_named_test(Vector_test_vsetvl_vadd            Vector_test big_core.out -c test_cores/test_big_core_full.yaml --input-file vsetvl_vadd.json)
-sparta_named_test(Vector_test_vsetvli_vadd_sew_32    Vector_test big_core.out -c test_cores/test_big_core_full.yaml --input-file vsetvli_vadd_sew_32.json)
-sparta_named_test(Vector_test_vsetvli_vl_max_setting Vector_test big_core.out -c test_cores/test_big_core_full.yaml --input-file vsetvli_vl_max_setting.json)
-sparta_named_test(Vector_undisturbed_test            Vector_test big_core.out -c test_cores/test_big_core_full_8_decode.yaml --input-file undisturbed_checking.json)
-sparta_named_test(Vector_unsupported_test            Vector_test big_core.out -c test_cores/test_big_core_full_8_decode.yaml --input-file vrgather.json)
+sparta_named_test(Vector_test_vsetivli      Vector_test big_core.out -c test_cores/test_big_core_full.yaml --input-file vsetivli_vaddvv.json)
+sparta_named_test(Vector_test_vsetvli       Vector_test big_core.out -c test_cores/test_big_core_full.yaml --input-file vsetvli_vaddvv.json)
+sparta_named_test(Vector_test_vsetvl        Vector_test big_core.out -c test_cores/test_big_core_full.yaml --input-file vsetvl_vaddvv.json)
+sparta_named_test(Vector_test_vsetivli_tail Vector_test big_core.out -c test_cores/test_big_core_full.yaml --input-file vsetivli_vaddvv_tail.json)
+#sparta_named_test(Vector_test_multiple_vset Vector_test big_core.out -c test_cores/test_big_core_full.yaml --input-file multiple_vset.json)
```

Comment:
@AaronGChan FYI this test wasn't being run before, but it was checked into the test repo so I enabled it. However, it currently fails in Rename so I need to look at it some more. Do you know why it's failing?

Reply:
Sorry, I think I forgot to uncomment it. I think it's failing due to timings, because I'm stopping the simulation at different time points to check the vset values. I guess a simpler solution would be to just split the test into multiple tests instead, that way we don't have to keep rerunning the simulation in the same test.

Reply:
I think we could just delete the test, and I could go in and add more tests related to it. I think with all the changes being added in recent PRs, the timings could have shifted by a cycle or two so debugging that older test probably isn't worth the effort.

Reply:
I updated the test to include 2 vsets that use different LMUL values followed by 2 `vadd.vv` instructions that will each generate a different number of uops. Then the tester can check that the correct nnumber of uops were generated. Not a perfect test, but it'll do for now.